apps/capacity-canvas/index.js

```diff
@@ -9,6 +9,23 @@ const syncChartToAnchor = canvas => {
       'a'
     ).href = `data:image/svg+xml;base64,\n${window.btoa(canvas.innerHTML)}`
   })
+
+  const removeByIndexButtons = document.querySelectorAll(
```

Comment:
Qui spezzerei in due la gestione. Sul componente `Board` gestirei il click sulle "x", la board poi emette un evento custom che `index.js` legge e invoca il cambio di dati. Inoltre a cosa ti serve l'attributo `listner-added`?

Reply:
mi torna, `listner-added` mi serve per non aggiungere N volte il listner